lib/screens/chatscreen/chat_components/chat_bubble.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:shareacab/screens/chatscreen/chat_models/chat_message.dart';
+import 'package:shareacab/screens/chatscreen/chat_modules/chat_detail_page.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+// ignore: must_be_immutable
+class ChatBubble extends StatefulWidget{
+  ChatMessage chatMessage;
+  ChatBubble({@required this.chatMessage});
+  @override
+  _ChatBubbleState createState() => _ChatBubbleState();
+}
+
+class _ChatBubbleState extends State<ChatBubble> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.only(left: 16, right: 16, top: 10, bottom: 10),
+      child: Align(
+        alignment: (widget.chatMessage.type == MessageType.Receiver ? Alignment
+            .topLeft : Alignment.topRight),
+        child: Container(
+          decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(30),
+            color: (widget.chatMessage.type == MessageType.Receiver ? Colors.grey : Colors.grey),
```

Comment:
Define the colors in main.dart and use them here. We want all the theming in one place